WordPress/wordpress-playground
# Match the .nvmrc node version to the changes made in commit ec2605b

## What is this PR doing?

It changes the minimum version of node in .nvmrc to the same version stated as a minimum [here](https://github.com/WordPress/wordpress-playground/commit/ec2605b#diff-7b99c1171afa5312d8c4adb6d14112de87c1776eaa38d93e32082d37fe27c42fR41) 

## What problem is it solving?

I was updating trunk, and ran `nvm use` and `npm install`. When running `npm install`, I got an error:

```
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: @php-wasm/compile@0.1.5
npm ERR! notsup Not compatible with your version of node/npm: @php-wasm/compile@0.1.5
```

## How is the problem addressed?

I changed the minimum version in the .nvmrc so that it matches the value stated [here](https://github.com/WordPress/wordpress-playground/commit/ec2605b#diff-7b99c1171afa5312d8c4adb6d14112de87c1776eaa38d93e32082d37fe27c42fR41)


## Testing Instructions

Run: 

- `nvm use`
- `nvm install`
- `npm run dev`

You should be able to run WP Playground and not get stuck on the `npm install` step with an error.
